### ISSUE TITLE
请问deepfm中二阶交叉是不是没对dense层进行交叉呀

### ISSUE THREAD

**Issue by downeykking** [shenweichen/DeepCTR-Torch#241]
Please refer to the [FAQ](https://deepctr-doc.readthedocs.io/en/latest/FAQ.html) in doc and search for the [related issues](https://github.com/shenweichen/DeepCTR-PyTorch/issues) before you ask the question.

**Describe the question(问题描述)**
A clear and concise description of what the question is.

**Additional context**
code如下
` def forward(self, X):

        sparse_embedding_list, dense_value_list = self.input_from_feature_columns(X, self.dnn_feature_columns,
                                                                                  self.embedding_dict)
        logit = self.linear_model(X)

        if self.use_fm and len(sparse_embedding_list) > 0:
            fm_input = torch.cat(sparse_embedding_list, dim=1)
            logit += self.fm(fm_input)

        if self.use_dnn:
            dnn_input = combined_dnn_input(
                sparse_embedding_list, dense_value_list)
            dnn_output = self.dnn(dnn_input)
            dnn_logit = self.dnn_linear(dnn_output)
            logit += dnn_logit

        y_pred = self.out(logit)

        return y_pred`

**Operating environment(运行环境):**
 - python version [e.g. 3.9]
 - torch version [e.g. 1.7.0,]
 - deepctr-torch version [e.g. 0.2.7,]

**Comment by ugirc** [comment]
是的，只对稀疏特征的Embedding向量进行二阶交叉

**Comment by downeykking** [comment]
> 

thanks~

**Comment by downeykking** [comment]
> 是的，只对稀疏特征的Embedding向量进行二阶交叉

thanks~
